pasta_eln/inputOutput.py

```diff
@@ -346,64 +312,66 @@ def processPart(part:dict[str,str]) -> int:
 ##########################################
 ###               EXPORT               ###
 ##########################################
-def exportELN(backend:Backend, projectID:str, fileName:str='', dTypes:list[str]=[], verbose:bool=True) -> str:
+def exportELN(backend:Backend, projectIDs:list[str], fileName:str, dTypes:list[str]=[], verbose:bool=False) -> str:
   """
   export eln to file
 
   Args:
     backend (backend): PASTA backend instance
-    projectID (str): docId of project
-    fileName (str): fileName which to use for saving; default='' saves in local folder
+    projectIDs (list): list of docIds of projects
+    fileName (str): fileName which to use for saving
     dTypes (list): list of strings which should be included in the output, alongside folders x0 & x1; empty list=everything is exported
     verbose (bool): verbose
 
   Returns:
     str: report of exportation
   """
-  # define initial information
-  docProject = backend.db.getDoc(projectID)
-  dirNameProject = docProject['-branch'][0]['path']
-  fileName = fileName or dirNameProject
-  fileName = fileName if fileName.endswith('.eln') else f'{fileName}.eln'
-  keysInSupplemental:set[str] = set()
-  filesNotInProject = []
-
-  def separate(doc: dict[str,Any]) -> tuple[str, dict[str,Any], dict[str,Any]]:
+  def separate(doc: dict[str,Any], dirNameProject_: str, filesNotInProject:list[str]) -> tuple[str, dict[str,Any]]:
     """ separate document into
     - main information (for all elns) and
     - supplemental information (specific to PastaELN)
+
+    Args:
+      doc (dict): document to separate into main and supplemental information
+      dirNameProject_ (str): name of the project directory
+      filesNotInProject (list): list of path-strings of files that are not in project folder hierarchy
```

Comment:
Did you mean an exclude list? The items to be excluded when exporting the project? 
If so, the following, would it not be more appropriate?

excludeItems (list): list of items to be excluded from the exported ELN file.

Reply:
I changed the code to make it more clear